builtin/mainmenu/content/dlg_package.lua

```diff
@@ -232,6 +233,53 @@ local function get_formspec(data)
 		else
 			table.insert_all(formspec, {"label[2,2;", fgettext("Loading..."), "]"} )
 		end
+	elseif current_tab == 4 then
+		if not package.releases and not data.releases_error and not data.releases_loading then
+			data.releases_loading = true
+
+			contentdb.get_package_releases(package, function(releases)
+				if not releases then
+					data.releases_error = true
+				end
+				ui.update()
+			end)
+		end
+
+		local function markup(text)
+			text = text:gsub("(%*%*)([^%s][^*]-[^%s])%*%*", "<b>%2</b>")
+			text = text:gsub("(%*)([^%s][^*]-[^%s])%*", "<i>%2</i>")
```

Comment:
```suggestion
			text = text:gsub("%*([^%s][^*]-[^%s])%*", "<i>%1</i>")
```

btw, pretty sure this will break on `***bolditalic***`. the first conversion will turn that into `<b>*bolditalic</b>*`. then it will become `<b><i>bolditalic</b></i>`. oops.